internal/command/user_v2_passkey.go

```diff
@@ -18,7 +17,7 @@ import (
 // RegisterUserPasskey creates a passkey registration for the current authenticated user.
 // UserID, usually taken from the request is compared against the user ID in the context.
 func (c *Commands) RegisterUserPasskey(ctx context.Context, userID, resourceOwner, rpID string, authenticator domain.AuthenticatorAttachment) (*domain.WebAuthNRegistrationDetails, error) {
-	if err := authz.UserIDInCTX(ctx, userID); err != nil {
+	if err := c.checkPermissionUpdateUser(ctx, resourceOwner, userID); err != nil {
```

Comment:
looks like we already have some checks in other parts (e.g. addHumanWebAuthN and createHumanTOTP) where we check for `user.credential.write` instead of `user.write`

also the linked issue  mentions TOTP but i can't find any changes in this PR related to that

Reply:
True, will change that.
Can't really tell you why there is TOTP mentioned, maybe it's also linked to an old ZITADEL version where there was the same topic there as well.